SILKIT-1608: disable nagle algorithm by default (#181)
vectorgrp/sil-kit: docs/configuration/middleware-configuration.rst

       By default, only a single connect is attempted.

   * - TcpNoDelay

>      - Disable the TCP_NODELAY flag on TCP sockets. This enables Nagle's algorithm.

Author: Yes!

       By default, only a single connect is attempted.

   * - TcpNoDelay

>      - Is used to set the TCP_NODELAY flag on TCP sockets. Setting this to true will disable Nagle's algorithm.

Reviewer: ```suggestion
     - Setting this to ``true`` will set the ``TCP_NODELAY`` flag on TCP sockets, which disables Nagle's algorithm.
```
